packages/flutter_tools/test/devfs_test.dart

```diff
@@ -359,8 +359,14 @@ class MockVMService extends BasicMock implements VMService {
   VM get vm => _vm;
 
   Future<Null> setUp() async {
-    _server = await HttpServer.bind(InternetAddress.LOOPBACK_IP_V6, 0);
-    _httpAddress = Uri.parse('http://[::1]:${_server.port}');
+    try {
+      _server = await HttpServer.bind(InternetAddress.LOOPBACK_IP_V6, 0);
+      _httpAddress = Uri.parse('http://[::1]:${_server.port}');
+    } on SocketException {
```

Comment:
`OSError` values errno etc. values differ across platforms. Feels like a bit of a rat's nest to deal with them if we can avoid it. For a test this seems fine.